config/express.js

```diff
@@ -45,10 +46,13 @@ module.exports = (app, config) => {
       ],
       styleSrc: [
         '\'self\'',
+        '\'unsafe-inline\'',
         'fast.fonts.net',
+        'https://dhrlmnmyf2njb.cloudfront.net/',
```

Comment:
@st3v3nhunt just spotted this after looking at #167 

With this URL it would be better to store it in an environment variable as this could well change per environment or at any time if we change where the font/assets are served from